README.md

```diff
@@ -67,6 +67,13 @@ using another playbook (for example):
 
 `ansible-playbook -e @config/openhpc.yml -i ansible/inventory_openhpc --vault-password-file=vault-password ansible/openhpc.yml`
 
+To setup and mount hyperconverged glusterfs:
+
+```
+ansible-playbook -e @config/openhpc.yml -i ansible/inventory_openhpc --vault-password-file=vault-password ansible/gluster_cluster.yml
+ansible-playbook -e @config/openhpc.yml -i ansible/inventory_openhpc --vault-password-file=vault-password ansible/mount_gluster.yml
```

Comment:
similarly, mount-glusterfs (although gluster-client is preferable)

Reply:
Done.